test/runtime/lb.go

```diff
@@ -301,6 +301,103 @@ var _ = Describe("RuntimeLB", func() {
 		status = vm.ContainerExec("client", testCmd)
 		status.ExpectSuccess("LB is not working after restart")
 	})
+
+	Context("Services Policies", func() {
+		BeforeEach(func() {
+			vm.PolicyDelAll()
```

Comment:
Should all services also be deleted in `BeforeEach`? Would ensure that there's no extra cruft lying around beforehand.